hubspot/connection/__init__.py

```diff
@@ -228,11 +230,11 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._session.close()
 
 
-_AuthenticationKey = Record.create_type('_AuthenticationKey', 'key_value')
+_AuthenticationKey = Record.create_type(str('_AuthenticationKey'), str('key_value'))
```

Comment:
Ok, I found the problem here, by doing `from __future__ import unicode_literals` every string becomes unicode which causes that problem when in `pyrecord` we pass the name into `type()`. I'm currently trying to solve this so we can merge your code, thank you for your patience :smiley: